lib/routes/data/styles.js

```diff
@@ -63,4 +63,10 @@ module.exports = {
       ],
     ],
   },
+  alertData: [
```

Comment:
should not be an array, or if it is, each object should be called separately below.